src/server/db/schema.sql

```diff
@@ -1,18 +1,7 @@
 -- from root directory -- psql < src/server/db/schema.sql
 
--- DROP DATABASE IF EXISTS fidgetywidget;
--- CREATE DATABASE fidgetywidget;
 \c fidgetywidget;
 
--- DROP TABLE IF EXISTS users_teams;
--- DROP TABLE IF EXISTS users_daily_activities;
--- DROP TABLE IF EXISTS users_incentives;
--- DROP TABLE IF EXISTS challenges;
--- DROP TABLE IF EXISTS users;
--- DROP TABLE IF EXISTS daily_activities;
--- DROP TABLE IF EXISTS incentives;
--- DROP TABLE IF EXISTS teams;
-
```

Comment:
keep the drop tables command so if we have to reseed if any of the table properties change we can